aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -128,6 +129,9 @@ impl From<FeatureFlag> for AptosFeatureFlag {
             FeatureFlag::ResourceGroups => AptosFeatureFlag::RESOURCE_GROUPS,
             FeatureFlag::MultisigAccounts => AptosFeatureFlag::MULTISIG_ACCOUNTS,
             FeatureFlag::DelegationPools => AptosFeatureFlag::DELEGATION_POOLS,
+            FeatureFlag::Ed25519PkValidateNoAbortOnWrongLength => {
+                AptosFeatureFlag::ED25519PKVALIDATENOABORTONWRONGLENGTH
```

Comment:
I think you will need to add `_` between the words here. Otherwise, a lint seems to be failing.

Reply:
And this is not readable at all

Reply:
Changed the flag to what I think is a better name and added the underscores

Reply:
I see the new name is `PUBKEY_VALIDATE_RETURN_FALSE_WRONG_LENGTH`. Can you add `ED25519_` as a prefix? Otherwise, it's not clear what the scope is. We have many modules with pubkey validation. For example, there was recently a feature flag for enabling a MultiEd25519 pubkey validation bugfix.

Reply:
Just added the prefix